### ISSUE TITLE
Missing first two words of any message

### ISSUE THREAD

**Issue by controller999** [rt400/ReversoTTS-HA#13]
I have Reverso TTS installed on my home assistant and it is working perfectly on all my google speakers and displays apart from one google display, no matter what i do it always misses off the first two words (if there is only two words it says nothing). I've tried using this with developer tools, automations and scrips but still nothing. I know there are ways around this but it is bothering me, have you come across this before? This device works perfect with google translate tts

**Comment by rt400** [comment]
Sorry for the delay..
Does this problem still exists ?
Can you share your config to let me try it to in my system ?

**Comment by cureheaded** [comment]
I have the same problem. In my experience and test it does not seem to be related to reverso but to Google speakers getting SLOOOOOOOWER en slower. I solved it by starting the messages with ". . . ".
controller999: use another output device to test this. I used the browser_mod for this
more info: i noticed a difference between the google hub and the mini in waking up speed. when the hub is 'awake' the message is fully spoken
